### DISCUSSION TITLE
user invitation with temporary password(expiring password)

### DISCUSSION THREAD

**Discussion by TinsG** [supabase/supabase#13805] [Questions] [answered]
Hi,

I am working on a project and there is this module where the system can invite user with email and on the invitation there should be an expiring temporary password attached and sent to the user. and the user should be able to login with the temporary password and change the password. but on supabase i have trouble inviting user with a temporary password. are there any method i should use?
so far i try `supabase.auth.admin.inviteUserByEmail()`

**Comment by GaryAustin1** [comment]
There is no temporary password solution built in.

**Reply by GaryAustin1** [reply, marked as answer]
You could send a magic link and then only have password login available on the site after that.

**Reply by TinsG** [reply]
Yes, thanks!
